providers/capvcd/standard/capvcd_suite_test.go

```diff
@@ -0,0 +1,65 @@
+package standard
+
+import (
+	"context"
+	"path"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	cr "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/clustertest"
+	"github.com/giantswarm/clustertest/pkg/application"
+	"github.com/giantswarm/clustertest/pkg/logger"
+	"github.com/giantswarm/clustertest/pkg/organization"
+	"github.com/giantswarm/clustertest/pkg/utils"
+	"github.com/giantswarm/clustertest/pkg/wait"
+)
+
+const KubeContext = "capvcd"
+
+var (
+	framework *clustertest.Framework
+	cluster   *application.Cluster
+)
+
+func TestCAPVCDStandard(t *testing.T) {
+	var err error
+	ctx := context.Background()
+
+	logger.LogWriter = GinkgoWriter
+
+	framework, err = clustertest.New(KubeContext)
+	if err != nil {
+		panic(err)
+	}
+
+	cluster = application.NewClusterApp(utils.GenerateRandomName("t"), application.ProviderCloudDirector).
```

Comment:
Which version is used here? How are we supposed to run this test suite? Per PR, Per release or periodically? And where? Sorry for my dumb questions.

Reply:
> Which version is used here? 
When not specified, as with this, then the latest version is retrieved from GitHub. I've [just merged a PR](https://github.com/giantswarm/clustertest/pull/15) that will allow for overriding that also with an environment variable (e.g. so a CI pipeline can pass it in) but I haven't updated this PR yet.

>  How are we supposed to run this test suite?
Currently it's only manual / locally following the instruction in the PR description. Once I have _something_ available to work with I can hook it up to a Tekton pipeline and work on getting it triggered automatically on PRs. I have a [WiP PR](https://github.com/giantswarm/tekton-resources/pull/41) waiting on this one to implement that but it's currently untested and will require this fixing - https://github.com/giantswarm/giantswarm/issues/26422.

The general idea is that these will run, at least initially, for the following situations:
* New PRs opened to this repo to test changes to the tests themselves.
* New PRs to the `cluster-X` apps
* New PRs to the `default-apps-X` apps

Following that we'll likely also trigger it on some form of cron trigger to try and catch mistakes made to the apps deployed to the MCs.

Eventually we want to get to a point where we can trigger some tests from other app repos too but we're still a long way off from that.